AL.ini

```
@@ -266,39 +269,36 @@ SQL1_1 = UPDATE vehicles SET blacklist = '0' WHERE id = ? AND pid = ?
 SQL1_INPUTS = 1, 2
 
 [updateVehicleBlacklistPlate]
-SQL1_1 = UPDATE vehicles SET blacklist = '1' WHERE pid = ? AND plate = ?
+SQL1_1 = UPDATE vehicles SET blacklist = '1' WHERE pid = ? AND id = ?
 SQL1_INPUTS = 1, 2
 
 [updateVehicleFuel]
-SQL1_1 = UPDATE vehicles SET active = '0', fuel = ?, damage = ? WHERE pid = ? AND plate = ?
+SQL1_1 = UPDATE vehicles SET active = '0', fuel = ?, damage = ? WHERE pid = ? AND id = ?
 SQL1_INPUTS = 1, 2, 3, 4
 
 [updateVehicleAll]
-SQL1_1 = UPDATE vehicles SET active = '0', inventory = ?, gear = ?, fuel = ?, damage = ? WHERE pid = ? AND plate = ?
+SQL1_1 = UPDATE vehicles SET active = '0', inventory = ?, gear = ?, fuel = ?, damage = ? WHERE pid = ? AND id = ?
 SQL1_INPUTS = 1, 2, 3, 4, 5, 6
 
 [updateVehicleGear]
-SQL1_1 = UPDATE vehicles SET gear = ? WHERE pid = ? AND plate = ?
+SQL1_1 = UPDATE vehicles SET gear = ? WHERE pid = ? AND id = ?
 SQL1_INPUTS = 1, 2, 3
 
 [updateVehicleTrunk]
-SQL1_1 = UPDATE vehicles SET inventory = ? WHERE pid = ? AND plate = ?
+SQL1_1 = UPDATE vehicles SET inventory = ? WHERE pid = ? AND id = ?
 SQL1_INPUTS = 1, 2, 3
 
 [insertVehicle]
-SQL1_1 = INSERT INTO vehicles (side, classname, type, pid, alive, active, inventory, color, plate, gear, damage) VALUES (?, ?, ?, ?, '1','1','[[],0]', ?, ?,'[]','[]')
+SQL1_1 = INSERT INTO vehicles (side, classname, type, pid, alive, active, inventory, color, plate, gear) VALUES (?, ?, ?, ?, '1','1','[[],0]', ?, ?,'[]'); SELECT LAST_INSERT_ID()
```

Use a separate statement.


```suggestion
SQL1_1 = INSERT INTO vehicles (side, classname, type, pid, alive, active, inventory, color, plate, gear) VALUES (?, ?, ?, ?, '1','1','[[],0]', ?, ?,'[]')
SQL1_2 = SELECT LAST_INSERT_ID()
```